io/io/inc/TFile.h

```diff
@@ -299,6 +299,11 @@ class TFile : public TDirectoryFile {
    static TFile       *Open(const char *name, Option_t *option = "",
                             const char *ftitle = "", Int_t compress = ROOT::RCompressionSetting::EDefaults::kUseCompiledDefault,
                             Int_t netopt = 0);
+
+   static std::unique_ptr<TFile> OpenU(const char *name, Option_t *option = "",
```

Comment:
```suggestion
   static std::unique_ptr<TFile> OpenUniquePtr(const char *name, Option_t *option = "",
```

Reply:
We shall have a naming discussion in Mattermost's I/O channel. I promise that nobody will want to spell `OpenUniquePtr` :-) And I don't want to see `Bronch`-style `Opan` either :-)

Reply:
@Axel-Naumann @pcanal Is it fine if we keep it as OpenU for the time being?